tests/TestUTestMacro.cpp

```diff
@@ -546,8 +546,8 @@ static void _failingTestMethodWithBYTES_EQUAL()
 TEST(UnitTestMacros, FailureWithBYTES_EQUAL)
 {
     fixture.runTestWithMethod(_failingTestMethodWithBYTES_EQUAL);
-    CHECK_TEST_FAILS_PROPER_WITH_TEXT("expected <97 0x61>");
-    CHECK_TEST_FAILS_PROPER_WITH_TEXT("but was  <98 0x62>");
+    CHECK_TEST_FAILS_PROPER_WITH_TEXT("expected <97 (0x61)>");
+    CHECK_TEST_FAILS_PROPER_WITH_TEXT("but was  <98 (0x62)>");
```

Comment:
@basvodde normally I can't wait for my stuff to be merged. But next time there's a situation like this one I'll be sure to put in some stupid compiler error so it can't be merged :-P